### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
- FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen sami.mikael.hiltunen@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
Please rebase :shipit:

**Comment by SamiHiltunen** [comment]
@jirikuncar rebased

**Comment by jirikuncar** [comment]
@SamiHiltunen I have discovered few more issues:
- [x] missing `sudo: false` in `.travis.yml`
- [x] missing `coveralls` package on Travis

**Comment by SamiHiltunen** [comment]
@jirikuncar fixed the problems you discovered

**Comment by SamiHiltunen** [comment]
@jirikuncar fixed the points you brought up

File: tests/test_messages.py [hunk 1]
```diff
@@ -19,16 +19,18 @@
...
-    "$Id$"
+from invenio_base.wrappers import lazy_import
 
+from invenio_testing import InvenioTestCase
 
-from invenio_base.wrappers import lazy_import
-from invenio.testsuite import make_test_suite, run_test_suite, InvenioTestCase
 webmessage_mailutils = lazy_import('invenio.utils.mail')
 
+__revision__ = \
```
Comment (jirikuncar) [inline tests/test_messages.py]: you can remove this legacy part

**Comment by SamiHiltunen** [comment]
If you need this for old release, please ping me.

**Pull request closed without merging**